### PR TITLE
[TECHNICAL-SUPPORT] LPS-99857

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @joshuacords

### CODE REVIEW

**Comment by jonathanmccann** [comment]
ci:test:sf

**Comment by jonathanmccann** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Hey @joshuacords  @jonathanmccann,

Could you please check if the broken tests in :apps:journal:journal-test:testIntegration are related with your code?

Thanks!! 
Eudaldo

**Comment by ealonso** [comment]
ci:test:relevant

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java [hunk 1]
```diff
@@ -6930,7 +6930,9 @@ else if (oldStatus == WorkflowConstants.STATUS_APPROVED) {
 
 			// Subscriptions
 
-			if ((article.getVersion() == 1.0) &&
+			if (article.equals(
```
Comment (ealonso) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java]: Hey @joshuacords,
It seems like getOldestArticle is the same as isLatestVersion, it is true?
Regards,
Eudaldo.

Comment (joshuacords) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java]: While getOldestArticle is not the same as isLatestVersion, looking at it now I don't see the need of isLatestVersion. If the article is the oldest, we know it should be "added".

**Comment by joshuacords** [comment]
Hi @ealonso, I'll send you a new PR with the removal of isLatestVersion change.

**Comment by joshuacords** [comment]
Resent here: https://github.com/ealonso/liferay-portal/pull/2512

**Comment by joshuacords** [comment]
ci:close

**Pull request closed without merging**